action.go

```diff
@@ -337,6 +337,8 @@ func (fa *FlowAction) Run(ctx context.Context, inputs map[string]interface{}, ha
 		} else if inst.Status() == model.FlowStatusFailed {
 			logger.Infof("Instance [%s] Failed", inst.ID())
 		}
+		// Release memory
+		inst = nil
```

Comment:
Not sure why this would make a difference, shouldn't that inst object be released when the go func ends?